### DISCUSSION TITLE
Is there a way to custom lsp load order?

### DISCUSSION THREAD

**Discussion by YuCao16** [mason-org/mason.nvim#360] [Q&A]
I'm using pyright and jedi at the same time, but they provide different document hover (and more), sometimes pyright on the top of lspinfo so document hover come from pyright, sometimes jedi. Is there a way I can control the order of lsp loading, for example load pyright before jedi?

**Comment by williamboman** [comment]
Hello! I assume you use [lspconfig](https://github.com/neovim/nvim-lspconfig) to configure servers? Mason is not directly involved in managing LSP servers, I'd recommend checking there for a potential solution.
